<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CON-DESKTOP03" ignored="0" name="FirefoxTest" tests="1" failures="0" timestamp="15 Mar 2018 08:38:31 GMT" time="21.853" errors="0">
  <testcase name="@BeforeTest setup" time="21.823" classname="testrunpck.RunningTest">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;unknown&amp;apos;, revision: &amp;apos;unknown&amp;apos;, time: &amp;apos;unknown&amp;apos;
System info: host: &amp;apos;CON-DESKTOP03&amp;apos;, ip: &amp;apos;172.17.61.18&amp;apos;, os.name: &amp;apos;Windows 8.1&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.3&amp;apos;, java.version: &amp;apos;1.8.0_121&amp;apos;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'CON-DESKTOP03', ip: '172.17.61.18', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_121'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:658)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:236)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:137)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:191)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:108)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:104)
	at testrunpck.RunningTest.setup(RunningTest.java:35)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException
	at org.openqa.selenium.remote.service.DriverService.waitUntilAvailable(DriverService.java:192)
	at org.openqa.selenium.firefox.XpiDriverService.start(XpiDriverService.java:94)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:78)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:637)
	... 25 more
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase name="testParameterWithXML" time="0.001" classname="testrunpck.RunningTest">
    <skipped/>
  </testcase> <!-- testParameterWithXML -->
</testsuite> <!-- FirefoxTest -->
